alpheios_nemo_ui/data/assets/js/passage-search.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * this function does an ajax prefetch of a requested passage identifier
+ * for the jump_passage input form which contains an input box for the user
+ * to enter the passage they want to go to. Preloading all of the possible
+ * passage identifiers is too performance intensive so we can't validate the
+ * user's input before loading the new page other than to try to load the passage
+ * itself. If the passage is found, we can go ahead and submit the form to load
+ * the requested page
+ */
+function tryPassage(e) {
+  e.preventDefault();
+  const el = document.querySelector("#find_subreference")
+  let ref = el.value
+  if (!ref || !ref.match(/^\d+(\.\d*)*$/)) {
+    console.warn('It is not a valid passage format! Try to use numbers and points.')
+    showError(`It is not a valid passage format - ${ref}! Try to use numbers and points.`);
```

Comment:
Let's just change this to "Please enter a valid passage reference."

Reply:
fixed